src/modules/module1.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-plusplus */
+const home = document.getElementById('home_link');
+
+const LOCAL_STORAGE_LIST_KEY = 'card.list';
+const cards = JSON.parse(localStorage.getItem(LOCAL_STORAGE_LIST_KEY)) || [];
+
+const clearElement = ((element) => {
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+});
+
+const save = (() => {
+  localStorage.setItem(LOCAL_STORAGE_LIST_KEY, JSON.stringify(cards));
+});
+
+home.addEventListener('click', () => {
+  async function getCards() {
+    const response = await fetch('https://db.ygoprodeck.com/api/v7/cardinfo.php?archetype=Blue-Eyes');
+    const allcards = await response.json();
+    const cardData = allcards.data;
+    cards.length = 0;
+    cards.push(...cardData.slice(0, 12));
+    save();
+  }
+  const result = getCards();
+  return result;
+});
+
+const displayElements = (() => {
+  const blueEyes = document.getElementById('card_list');
+  clearElement(blueEyes);
+  for (let i = 0; i < cards.length; i++) {
+    const blueEye = document.getElementById('card_list');
```

Comment:
The declared `blueEyes` variable on line 31 can be used here, its the same element :ok_hand: